### PR TITLE
Added hand delivery flag to dto and model

### PR DESCRIPTION
**Pull request by ryangrundy7**
# Motivation and Context

- Added hand delivery to case container dto.

# How to test?

 
- Run with other branches and test with acceptance tests.

# Links

https://trello.com/c/774V6fGU/580-add-handdelivery-flag-to-casecreated-and-caseupdated-events-and-case-api-8
# Screenshots (if appropriate):

### CODE REVIEW

**Comment by ryangrundy7** [comment]
[add handDelivery flag to caseCreated and caseUpdated events and case API (8)](https://trello.com/c/774V6fGU/580-add-handdelivery-flag-to-casecreated-and-caseupdated-events-and-case-api-8)

File: src/main/java/uk/gov/ons/census/caseapisvc/model/dto/CaseContainerDTO.java [hunk 1]
```diff
@@ -63,4 +63,6 @@
   private String lad;
 
   private List<CaseEventDTO> caseEvents;
+
+  private Boolean handDelivery;
```
Comment (AdamHawtin) [inline src/main/java/uk/gov/ons/census/caseapisvc/model/dto/CaseContainerDTO.java]: If we're never sending `null` or missing this value off can we make this a `boolean` instead?

File: src/main/java/uk/gov/ons/census/caseapisvc/model/entity/Case.java [hunk 2]
```diff
@@ -114,4 +114,7 @@
   @Column(columnDefinition = "timestamp with time zone")
   @UpdateTimestamp
   private OffsetDateTime lastUpdated;
+
+  @Column(columnDefinition = "BOOLEAN")
```
Comment (AdamHawtin) [inline src/main/java/uk/gov/ons/census/caseapisvc/model/entity/Case.java]: Column definition shouldn't be necessary, same as case service

File: src/main/java/uk/gov/ons/census/caseapisvc/model/dto/CaseContainerDTO.java (hunk 1 above)
Comment (NickGrantONS) [reply src/main/java/uk/gov/ons/census/caseapisvc/model/dto/CaseContainerDTO.java]: we'd have to change the underlying DB type too, or make it non-nullable

Comment (NickGrantX) [reply src/main/java/uk/gov/ons/census/caseapisvc/model/dto/CaseContainerDTO.java]: make this `boolean` not `Boolean` to match the underlying entity

**Review by NickGrantX — CHANGES_REQUESTED** [review]
What Adam said

File: src/main/java/uk/gov/ons/census/caseapisvc/model/entity/Case.java [hunk 3]
```diff
@@ -114,4 +114,6 @@
   @Column(columnDefinition = "timestamp with time zone")
   @UpdateTimestamp
   private OffsetDateTime lastUpdated;
+
+  private boolean handDelivery;
```
Comment (AdamHawtin) [inline src/main/java/uk/gov/ons/census/caseapisvc/model/entity/Case.java:119]: Needs `@Column(nullable = false, columnDefinition = "BOOLEAN DEFAULT false")` to match the DDL

**Review by AdamHawtin — CHANGES_REQUESTED** [review]

**Review by AdamHawtin — APPROVED** [review]

File: src/main/java/uk/gov/ons/census/caseapisvc/model/entity/Case.java [hunk 4]
```diff
@@ -114,4 +114,7 @@
   @Column(columnDefinition = "timestamp with time zone")
   @UpdateTimestamp
   private OffsetDateTime lastUpdated;
+
+  @Column(nullable = false, columnDefinition = "BOOLEAN DEFAULT false")
```
Comment (NickGrantX) [inline src/main/java/uk/gov/ons/census/caseapisvc/model/entity/Case.java:118]: `nullable = false` is implied, but not going to block it on such a minor point

**Review by NickGrantX — APPROVED** [review]
`nullable = false` is implied on any `boolean` value, but I'm not going to block it on such a minor point. Looks good 👍

**Pull request merged**